### DISCUSSION TITLE
Where is `extend_search_next` and `extend_search_prev` ?

### DISCUSSION THREAD

**Discussion by NNBnh** [helix-editor/helix#1172] [Q&A] [answered]
At the time of writing in [Kemap doc](https://docs.helix-editor.com/keymap.html):

| Key          | Description                                 | Command            |
| ------------ | ------------------------------------------- | ------------------ |
| <kbd>/</kbd> | Search for regex pattern                    | `search`           |
| <kbd>?</kbd> | Search for previous pattern                 | `rsearch`          |
| <kbd>n</kbd> | Select next search match                    | `search_next`      |
| <kbd>N</kbd> | Select previous search match                | `search_prev`      |
| <kbd>*</kbd> | Use current selection as the search pattern | `search_selection` |

What happen to the `extend_search_next` and `extend_search_prev`, it's be forgot after [_"Add reverse search functionality"_](https://github.com/helix-editor/helix/commit/911b9b3276cb155eab023b24f1a6f336f4054087)? It's seem to be temporarily removed right?

Could we add it back to <kbd>alt-n</kbd>, <kbd>alt-N</kbd> or <kbd>H</kbd>, <kbd>L</kbd> ?

**Comment by archseer** [comment, marked as answer]
It's under select mode. So `vnnnnn` etc.

**Reply by NNBnh** [reply]
Nice, what a great design!
